src/cli.rs

```diff
@@ -36,12 +44,14 @@ impl Cli {
         let mut coordinator = Coordinator::default();
         coordinator.add_target(&rbt.default);
 
-        let runner = crate::fake_runner::FakeRunner::default();
+        let runner: Box<dyn crate::coordinator::Runner> = if self.use_fake_runner {
+            Box::new(crate::fake_runner::FakeRunner::default())
```

Comment:
Why box over a trait/generic?

Reply:
it's a box containing a trait—the two implementations have different sizes, so if they're unboxed the compiler complains about needing to relax the implied `Sized` trait bound. Searching around, it seemed like the recommended approach was just to box the trait.

That said, I think the fake runner is not long for this world… it does not produce files in the way that we need in later PRs, so it's not actually that useful for testing.

Reply:
Ack.